<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>简单ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushBtnInitList">
        <property name="text">
         <string>初始化列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushBtnClearList">
        <property name="text">
         <string>清除列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxIsEdit">
        <property name="text">
         <string>可编辑</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QComboBox" name="comboBoxCity">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>有用户数据的ComboBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushBtnInitCityAreaCode">
        <property name="text">
         <string>初始化城市+区号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCityAreaCode">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>兰州</string>
         </property>
         <property name="icon">
          <iconset resource="imgs.qrc">
           <normaloff>:/icon/imgs/city.png</normaloff>:/icon/imgs/city.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>呼和浩特</string>
         </property>
         <property name="icon">
          <iconset resource="imgs.qrc">
           <normaloff>:/icon/imgs/city.png</normaloff>:/icon/imgs/city.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>长沙</string>
         </property>
         <property name="icon">
          <iconset resource="imgs.qrc">
           <normaloff>:/icon/imgs/city.png</normaloff>:/icon/imgs/city.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>福州</string>
         </property>
         <property name="icon">
          <iconset resource="imgs.qrc">
           <normaloff>:/icon/imgs/city.png</normaloff>:/icon/imgs/city.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>QPlainTextEdit（具有标准快捷菜单）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushBtnAddTextToComboBox">
        <property name="text">
         <string>将文本内容添加到ComboBox</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushBtnClearText">
        <property name="text">
         <string>清除文本内容</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxIsWrite">
        <property name="text">
         <string>Read Only</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
